app/react-native/package.json

```diff
@@ -40,7 +40,7 @@
     "babel-plugin-transform-regenerator": "^6.26.0",
     "babel-plugin-transform-runtime": "^6.23.0",
     "babel-preset-env": "^1.7.0",
-    "babel-preset-minify": "^0.5.0 || 0.6.0-alpha.5",
+    "babel-preset-minify": "^0.6.0-alpha.5 || 0.6.0-alpha.5",
```

Comment:
@dependabot-bot this change looks weird to me

Reply:
That definitely looks wrong! I'll take a look.

Reply:
Looks like you've got `increase versions` specified as the update strategy for Dependabot - what would you want to happen here? Should Dependabot remove the `||` to make this `0.6.0-alpha.5`? What if it was updating to `0.6.0-alpha.7`, or to `0.7.0`?

Reply:
I would expect `^0.5.0 || 0.6.0-alpha.7` in the former case, and just `^0.7.0` in the latter but it's better to ask @Stephanemw who originally wrote this constraint

Reply:
Thanks for catching this (apologies I only just got the ping). Ideally we want to generically depend on a stable version, and pull in a latest one if available. I added 0.6.0-alpha.5 as that's the latest available from babel-preset-minify that contains my fix vs. vulnerable lodash dependencies. Unfortunately the minify maintainers still haven't released this as stable.

During discussions on this patch, the desire was to avoid forcing users onto the alpha version.

@Hypnosphi is correct, a future version bump would hopefully go to ^0.6.0 or ^0.7.0.